lib/switchtec.c

```diff
@@ -56,6 +57,35 @@
  * @{
  */
 
+/* maximum firmware module string length */
+#define FW_MOD_NAME_LEN_MAX	32
+
+/* maximum number of log entries for each module */
+#define LOG_ENTRIES_MAX		512
+
+/* maximum log entry string length */
+#define LOG_ENTRY_LEN_MAX	512
```

Comment:
We really don't need all these defines at the top of the file. Most of them are only used in one place...

LOG_ENTRIES_MAX isn't even used.

Reply:
Yep I forgot to remove LOG_ENTRIES_MAX when changing things to dynamically allocate.

I'm used to our FW convention of grouping all #define's at the top of the file. I'll move them above the functions that use them.

Reply:
My point was you probably didn't need a define at all for most of these things...

Eg the line length can probably just be

`char line[512];`

and use `sizeof(line)` in the function, if it's necessary. No point in adding any kind of global define for it.

Reply:
Ok, I'll do that.